addons/mail/static/src/js/views/activity/activity_cell.js

```diff
@@ -0,0 +1,42 @@
+odoo.define("mail.ActivityCell", function (require) {
+"use strict";
```

Comment:
I think that the new convention (for new files) is to indent the first level as well (like it would naturally be)